Adds footprint for BC2003 2032 battery holder

isundaylee opened the pull request:
Adds footprint for Memory Protection Devices' BC2003 battery holder. It is a battery strap for 1xCR2032 battery. 

Datasheet: http://www.memoryprotectiondevices.com/datasheets/BC-2003-datasheet.pdf
Screenshot:

<img width="955" alt="screen shot 2018-05-04 at 1 42 40 pm" src="https://user-images.githubusercontent.com/1572861/39642888-0a030eb4-4fa1-11e8-8230-24f2b37dabe3.png">

Part image: 

<img src="https://media.digikey.com/Photos/Memory%20Protection%20Photos/MFG_BC-2003_sml.jpg">

------------

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the footprint(s) you are contributing
- [x] An example screenshot image is very helpful 
- [x] If there are matching symbol or 3D model pull requests, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required

poeschlr commented:
One measurement is not quite right (See screenshot):
![screenshot from 2018-05-05 03-36-49](https://user-images.githubusercontent.com/18327350/39658545-16174ccc-5016-11e8-9cf3-d985a4b47a6a.png)

isundaylee commented:
@poeschlr Good catch! My apology. I just fixed the dimension. Updated screenshot: 

![screen shot 2018-05-04 at 11 49 30 pm](https://user-images.githubusercontent.com/1572861/39659424-d8e0c678-4ff5-11e8-8d3f-dbd266e4b340.png)

poeschlr commented:
Thanks
